components/camel-test-spring/src/main/java/org/apache/camel/test/spring/CamelSpringBootExecutionListener.java

```diff
@@ -31,6 +33,7 @@ public void prepareTestInstance(TestContext testContext) throws Exception {
         LOG.info("@RunWith(CamelSpringBootJUnit4ClassRunner.class) preparing: {}", testContext.getTestClass());
 
         Class<?> testClass = testContext.getTestClass();
+        SpringCamelContext.setNoStart(true);
```

Comment:
Something similar is already done by the Camel Spring Test support (non Spring Boot) in [CamelSpringDelegatingTestContextLoader](https://github.com/apache/camel/blob/83ee9a5e58375e7bec8795e76f57db2fd6b5558c/components/camel-test-spring/src/main/java/org/apache/camel/test/spring/CamelSpringDelegatingTestContextLoader.java#L54-L59), so I think we're OK here.